modules/hal_sensors/sensor_differentialpressure/src/sensor_differentialpressure.hpp

```diff
@@ -78,6 +79,14 @@ class DifferentialPressureSensor : public UniversalSensor {
   int16_t sensor_raw_;
   int16_t conversion_factor_;
   uint8_t sensor_buffer_[kSdp810BufferSize];
+  uint16_t sampleTime;
+  double air_drag;
+  double hose_orifice_ratio;
+  double orifice_diameter;
+  double correction_factor;
+  double orifice_surface;
+  double volume;
+  double volume_;
```

Comment:
As said above by making many of these variables constants and placing them as defines in the source file can improve readability. Also think well about the names and trailing _ after the name :)